Sources/StreamChatSwiftUI/Generated/L10n.swift

```diff
@@ -5,527 +5,527 @@ import Foundation
 
 // MARK: - Strings
 
-internal enum L10n {
+public enum L10n {
```

Comment:
The text generation is internal to the SDK, integrating apps are only able to change the values of the texts via the localizable files.